packager/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Maybe switch to one of the Google-provided images? But seems to be go 1.8, not 1.10:
+# https://cloud.google.com/appengine/docs/flexible/custom-runtimes/build#base
+FROM golang:1.10
+
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+ENTRYPOINT ["dumb-init", "--"]
+
+RUN groupadd -r amppkg && useradd --no-log-init -r -g amppkg amppkg
+WORKDIR /amppkg
+
+COPY amppkg.toml /amppkg
+# amppkg is hard-coded to use /www as the docroot
+COPY dist /www
+COPY certs/privkey.pem.enc /amppkg
+COPY certs/cert.pem.enc /amppkg
+COPY decrypt.sh /amppkg
+RUN chmod +x /amppkg/decrypt.sh
+RUN chown -R amppkg:amppkg /amppkg
+
+# Because it's not possible to directly install a fork of a package via "go get"
+# (see https://blog.sgmansfield.com/2016/06/working-with-forks-in-go/), we
+# instead download the upstream, apply the appropriate patch (love love GitHub's
+# API!), and then install the patched version.
+RUN go get -d -u github.com/ampproject/amppackager/cmd/amppkg
+RUN curl -s https://github.com/ampproject/amppackager/compare/master...ithinkihaveacat:abe.patch | git -C $GOPATH/src/github.com/ampproject/amppackager apply
+RUN go install github.com/ampproject/amppackager/cmd/amppkg
+
+USER amppkg
+EXPOSE 8080
+CMD [ "bash", "-c", "./decrypt.sh && exec amppkg -config /amppkg/amppkg.toml" ]
```

Comment:
Where does the stdout from this cmd go? Does appengine store it somewhere for later perusal?